### PR TITLE
update version in the bms-support branch

### PR DESCRIPTION
**Pull request by alex-basinov**
## Description
Updating version to 2.41.0-bms

## Related issue
[b/267347168](http://b/267347168)

## Checklist:
- Unit tests
  - [x] Unit tests do not apply.
  - [ ] Unit tests have been added/modified and passed for this PR.
- Integration tests
  - [x] Integration tests do not apply.
  - [ ] Integration tests have been added/modified and passed for this PR.
- Documentation
  - [x] This PR introduces no user visible changes.
  - [ ] This PR introduces user visible changes and the corresponding documentation change has been made.
- Minor version bump
  - [ ] This PR introduces no new features.
  - [ ] This PR introduces new features, and there is a separate PR to bump the [minor version](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/VERSION) since the last [release](https://github.com/GoogleCloudPlatform/ops-agent/releases) already.
  - [x] This PR bumps the version.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-PKG_VERSION="2.41.0"
+PKG_VERSION="2.41.0-bms"
```
Comment (igorpeshansky) [inline VERSION:1]: This breaks the RPM builds with the following:
```
+ rpmbuild --define '_source_filedigest_algorithm md5' --define 'package_version 2.41.0-bms' --define '_sourcedir /tmp/cache_run/golang' --define '_rpmdir /tmp/cache_run/golang' -ba pkg/rpm/google-cloud-ops-agent.spec
error: line 16: Illegal char '-' in: Version: 2.41.0-bms
```

According to the [Fedora Packaging Guildelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version), only "ASCII letters (upper and lower case), digits and periods" are allowed in RPM version strings.

You might want to apply the following edit to `pkg/rpm/build.sh`:
```diff
-  --define "package_version $PKG_VERSION" \
+  --define "package_version ${PKG_VERSION//-/.}" \
```
or even:
```diff
-  --define "package_version $PKG_VERSION" \
+  --define "package_version ${PKG_VERSION//[^a-zA-Z0-9.]/.}" \
```
We should probably make the latter change in `master`…

Comment (igorpeshansky) [reply VERSION:1]: This is now done. Please pull in the changes from `master`.

Comment (igorpeshansky) [reply VERSION]: Actually, I just rebased `bms-support` (the base branch) on top of `master`. There were a couple of conflicts that I resolved. This should be good to go.

**Review by quentinmit — APPROVED** [review]

**Review by igorpeshansky — APPROVED** [review]
LGTM :shipit:

**Pull request merged**